spec/models/artist_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Artist, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+    describe 'Artist' do
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.